docker/Dockerfile

```
@@ -241,6 +242,7 @@ RUN --mount=type=cache,target=${CCACHE_DIR} \
   --mount=type=bind,source=src/universe/external/rtklib_ros_bridge,target=/autoware/src/universe/external/rtklib_ros_bridge \
   --mount=type=bind,source=src/universe/external/tier4_ad_api_adaptor,target=/autoware/src/universe/external/tier4_ad_api_adaptor \
   --mount=type=bind,source=src/universe/external/tier4_autoware_msgs,target=/autoware/src/universe/external/tier4_autoware_msgs \
+  --mount=type=bind,source=src/middleware/external/agnocast,target=/autoware/src/middleware/external/agnocast \
```

It’s easier to maintain if we align it with this line. https://github.com/autowarefoundation/autoware/pull/5917/files#diff-f34da55ca08f1a30591d8b0b3e885bcc678537b2a9a4aadea4f190806b374ddcR52

```suggestion
  --mount=type=bind,source=src/middleware/external,target=/autoware/src/middleware/external \
```